templates/gitserver/gitserver.Service.yaml

```diff
@@ -7,31 +7,9 @@ metadata:
 spec:
   clusterIP: None
   ports:
-  - name: unused
-    port: 10811
-    targetPort: 10811
+    - name: unused
```

Comment:
was this broken before?

Reply:
I'm not sure what you mean. If it's just the indentation, then VSCode/prettier formats it like this.

Reply:
Yes, I mean the formatting. Looks like it is equivalent in yaml, I just wanted to double check that the whitespace wasn't significant here

Reply:
I assume that it wasn't auto-formatted previously because of the syntax-errors introduced by the templating logic.

Reply:
this is different indentation to what kubernetes pretty prints out. but if CI is happy, this should be good (since we validate with kubeval)